src/NexusMods.App.UI/Pages/CollectionDownload/CollectionDownloadViewModel.cs

```diff
@@ -295,13 +306,16 @@ public CollectionDownloadViewModel(
                 }).AddTo(disposables);
 
             numDownloadedOptionalItemsObservable
+                .CombineLatest(hasInstalledAllOptionalItems)
                 .OnUI()
-                .Subscribe(numDownloadedOptionalItems =>
+                .Subscribe(tuple =>
                 {
+                    var (numDownloadedOptionalItems, hasInstalledAllOptionals) = tuple;
                     var hasDownloadedAllOptionalItems = numDownloadedOptionalItems == OptionalDownloadsCount;
 
                     CountDownloadedOptionalItems = numDownloadedOptionalItems;
-                    _canInstallOptionalItems.OnNext(numDownloadedOptionalItems > 0);
+                    HasInstalledAllOptionalItems.Value = hasInstalledAllOptionals;
+                    _canInstallOptionalItems.OnNext(numDownloadedOptionalItems > 0 && !hasInstalledAllOptionals);
```

Comment:
You should only be able to install all optional items if you have downloaded all optional items, not just one.

Reply:
Hm, I'm not sure actually. Since perhaps I would want to just download most of them but not all of them (or one of the external optional ones fails or something) and then I would still want to have the convenience of the button at the top.
It should dynamically update based on whether there are downloaded optional items that need to be installed or not.

Reply:
I think it already works as you said it should:
![image](https://github.com/user-attachments/assets/bfe8f46f-bb74-49d8-9ebc-4b174bdcc743)

Reply:
It's harder to do it that way, something we can do in a later PR. This PR should just have a button for install all optional items.

Reply:
Changed it. It was already behaving that way due to the IsVisible logic, so it didn't really matter, but I guess it's better to have consistent code.